en/micro-integrator/docs/setup/observability/setting-up-cloud-native-observability-in-kubernetes.md

```diff
@@ -79,6 +79,12 @@ The above steps deploy the observability solution with tracing capabilities and
 
 This deployment installs Jaeger-Operator. To install the Jaeger deployment, follow the steps in [Jaeger Operator documentation - Creating a new instance](https://github.com/jaegertracing/helm-charts/tree/master/charts/jaeger-operator#creating-a-new-jaeger-instance) and deploy the preferred Jaeger deployment.
 
+!!! Note
```

Comment:
Is this a sequence of steps? If not, we should have this as bullet points instead of a numbered list.

Reply:
No. These are some things to note before configuring Jaeger.

Reply:
Done